css/game.css

```diff
@@ -0,0 +1,230 @@
+}
+
+@font-face {
+    font-family: 'Heebo';
+    src: url('../assets/fonts/heebo/Heebo/Heebo-Regular.ttf');
+}
+
+h1, .game-over {
+    text-align: center;
+    font-family: 'Marker';
+    font-size: 4rem;
+}
+
+body {
+    background: #b8c77c;
+    font-family: 'Heebo';
+    font-size: 3rem;
+}
+
+
+
+/*=============================================
+=            HEADER                            =
+=============================================*/
+
+header {
+    background-image: url("../assets/imgs/whiteboard.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 80%;
+    height: 500px;
+    margin: 0 auto;
+    font-family: 'Marker';
+}
+
+.clock {
+    width: 15%;
+    position: relative;
+    top: 15px;
+    left: 50px;
+    display: inline-block;
+}
+
+.score {
+    width: 22%;
+    position: relative;
+    top: 15px;
+    display: inline-block;
+}
+
+.title {
+    position: relative;
+    top: 15px;
+    width: 60%;
+    display: inline-block;
+}
+
+.whiteboard {
+    width: 80%;
+    margin: 0 auto;
+}
+
+/* Keep the game-over screen hidden until the game clock has run down */
+.game-over {
+    display: none;
+}
+
+/* the countdown clock that appears centered on the screen when the game start */
+.countdown {
+    margin: 0 auto;
+    display: none;
+}
+
+/*=============================================
+=            Game Board                            =
+=============================================*/
+.board {
+    margin-top: -130px;
+    margin-left: 25px;
+    width: 100%;
+    position:relative;
+    z-index: 5;
+    text-align: center;
+}
+
+.desks {
+    display: inline-block;
+    margin-left: -35px;
+    width: 28%;
+    position: relative;
+    bottom: 30px;
+    margin-top: -110px;
+}
+
+.desks.middle {
+    bottom: 65px;
+}
+
+.table {
+    width: 83%;
+    position: relative;
+    z-index: 3;
+}
+
+.rat {
+    display: inline-block;
+    width: 49%;
+    position: relative;
+    z-index: 1;
+    top: 56px;
+}
+
+.ratImg {
+    width: 82%;
+}
+
+
+/*=============================================
+=           Splash Screen styles                           =
+=============================================*/
+
+.gamescreen {
+    height: 800px;
+    width: 1000px;
+    background-color: #9B9B9B;
+    margin: 0 auto;
+    margin-top: 100px;
+    border: solid 2px black;
+}
+
+.buttons {
+    text-align: center;
+    position: relative;
+    z-index: 10;
+}
+
+.image {
+    height: 750px;
+    z-index: 1;
+    position: absolute;
+    margin-left: 120px;
+    margin-top: 20px;
+}
+
+.heading {
+    font-size: 100px;
+    color: white;
+    transform: rotate(-20deg);
+    margin-top: 200px;
+    margin-left: 150px;
+    position: absolute;
+}
+.button {
+    background-color: #A0CED9;
+    border: none;
+    color: white;
+    padding: 20px;
+    text-align: center;
+    font-size: 25px;
+    cursor: pointer;
+    height: 75px;
+    width: 200px;
+    border-radius: 20px;
+    margin-top: 350px;
```

Comment:
i find it unlikely that all buttons need this css on. it should be on `.buttons`, keep .button generic and then you can use it instead of `.back` on line 214